tasks/icinga2-web2.yml

```diff
@@ -5,6 +5,44 @@
 - include_tasks: icinga2-web2-Debian.yml
   when: ansible_os_family == 'Debian'
 
+- name: IcingaWeb2 - Create configuration files
+  become: true
+  template:
+    src: "icingaweb2-config.ini.j2"
+    dest: "/{{ item.dest }}"
```

Comment:
Your `dest` below are already absolute.
Now, I might have used `dest: "/etc/icingaweb2/{{ item.dest }}"` here, but I can also see the advantage of having the absolute path below. I'd still refactor this and move the common root up here.